src/TransitorySensitivities.jl

```diff
@@ -149,15 +149,15 @@ end
 function normalizefulltransitorysensitivities!(dSdt,sim::Simulation,t)
     y = sim.sol(t)
     @views ns = y[sim.domain.indexes[1]:sim.domain.indexes[2]]
-    dSdt .*= sim.p'
-    @views dSdt[sim.domain.indexes[1]:sim.domain.indexes[2],:] ./= ns
+    dSdt[:,sim.domain.indexes[2]:end] .*= sim.domain.p[sim.domain.indexes[2]:end]'
+    @views dSdt[1:sim.domain.indexes[2],:] ./= ns
```

Comment:
Why do we change from `dSdt[sim.domain.indexes[1]:sim.domain.indexes[2],:] ./= ns` to `dSdt[1:sim.domain.indexes[2],:] ./= ns`? This seems wrong. The `ns` dimension is `sim.domain.indexes[1]:sim.domain.indexes[2]`?

Reply:
I think for single domain simulation sim.domain.indexes[1] is always one for the current version

Reply:
Also why `dSdt[:,sim.domain.indexes[2]:end] .*= sim.domain.p[sim.domain.indexes[2]:end]'`? We are trying to multiply the dSdt of rates with their p, right? Shouldn't it be `dSdt[:,sim.domain.indexes[2]+1:end] .*= sim.domain.p[sim.domain.indexes[2]+1:end]'`? Isn't `sim.domain.indexes[2]` the index of the last species?

Reply:
This may be the result of a mixup in the rebase...I think I had to fix these issues at the end, because we have a  test  that involves calculating transitory sensitivities of a Simulation that is within a SystemSimulation where 1 will be incorrect.

Reply:
I've squashed in the commit at the end now the first commit should look reasonable now.